### DISCUSSION TITLE
Process for new updates

### DISCUSSION THREAD

**Discussion by ted-miller** [Yaskawa-Global/motoros2_interfaces#18] [General]
@gavanderhoorn With the automation you're putting in place, what is the process for updating motoros2_interfaces such that it gets propagated into the library and consumable by motoros2?

**Comment by gavanderhoorn** [comment]
The idea is to eventually either:

- support running MotoROS2 CI against, potentially automatically built and uploaded, draft releases of `micro_ros_motoplus` (like [this](https://github.com/Yaskawa-Global/motoros2/actions/runs/9681673027/job/26712796460?pr=227)), or
- extend MotoROS2 CI to accept either a configuration file (some YAML in a GitHub Gist fi) or arguments (like `Yaskawa-Global/motoplus_libmicroros_builder`) such that we can instruct it to build both MotoROS2 *and* a specified version of `micro_ros_motoplus` (from a fork fi)

we'd have to be a bit careful about not letting just anyone do this (access to secrets, etc), but that should make it possible to push changes to messages to a branch, changes to MotoROS2 to another and then have CI check it all.

We're not there yet though.
